### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by WilliamDavidHarrison**
### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [x] This is Software as a Service not self hosted
 - [x] It has a free tier not just a free trial
 - [x] The submission mentions what is free
 - [x] The submission is not already present in the list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 [![Track Awesome List](https://www.trackawesomelist.com/badge.svg)](https://www.trackawesomelist.com/ripienaar/free-for-dev)
 
-*NOTE:* This list is only for as-a-Service offerings, not for self-hosted software. For a service to be eligible it has to offer a free tier and not just a free trial. If the free tier is time-bucketed it has to be for at least a year. We also consider the free tier from a security perspective, so SSO is fine but I will not accept services that restrict TLS to paid-only tiers.
+**NOTE**: This list is only for as-a-Service offerings, not for self-hosted software. For a service to be eligible it has to offer a free tier and not just a free trial. If the free tier is time-bucketed it has to be for at least a year. We also consider the free tier from a security perspective, so SSO is fine but I will not accept services that restrict TLS to paid-only tiers.
 
-Table of Contents
+**Table of Contents**
```
Comment (ripienaar) [inline README.md:15]: I don’t think this is needed. The == under it already does this?

Comment (WilliamDavidHarrison) [reply README.md:15]: Oh

Comment (ripienaar) [reply README.md:15]: Well more accurate to say it makes it an H1 that’s sufficiently bold and big and in style with the rest of the TOC

**Comment by WilliamDavidHarrison** [comment]
Removed **** at table off contents

**Comment by ripienaar** [comment]
Thank you

**Pull request merged**